src/core/packages/rendering/server-internal/src/bootstrap/bootstrap_renderer.ts

```diff
@@ -58,7 +62,11 @@ export const bootstrapRendererFactory: BootstrapRendererFactory = ({
 
   return async function bootstrapRenderer({ uiSettingsClient, request, isAnonymousPage = false }) {
     let darkMode: DarkModeValue = false;
-    let themeName: string = 'amsterdam';
+    let themeName: string = DEFAULT_THEME_NAME;
+
+    if (packageInfo.buildFlavor !== 'serverless') {
+      themeName = 'borealis';
+    }
```

Comment:
I wonder if it is worth having this conditional also depend on some configuration setting... That way when the time comes we can rollout borealis incrementally to serverless. Not a blocker for this PR, but would be nice to hear your thoughts!

Reply:
I personally would want this to depend _solely_ on a configuration setting.